packages/sources/synthetix-debt-pool/src/commons.ts

```diff
@@ -0,0 +1,67 @@
+import { Validator, Requester, Logger } from '@chainlink/ea-bootstrap'
+import { AdapterRequest, AdapterResponse, InputParameters } from '@chainlink/types'
+import { BigNumber, ethers } from 'ethers'
+import { SupportedChains, Config } from './config'
+import { AdapterError } from '@chainlink/ea-bootstrap'
+import { SynthetixJS, synthetix } from '@synthetixio/contracts-interface'
+
+const inputParameters: InputParameters = {
+  chainSources: {
+    required: false,
+    description: `Array of chains to pull debt from. Options for array elements are "mainnet", "mainnet-ovm", "kovan", "kovan-ovm"`,
+    type: 'array',
+  },
+}
```

Comment:
In order for this to work with the doc generator scripts, we need to export this from each endpoint. Would it be ok to export this from each endpoint somehow? Either copying the object in each, or maybe import and re-export?

Reply:
sure I think we can do that